TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -289,6 +289,17 @@ public static void OnCodeElement(StartStatement statement, CodeElementsParser.St
         }
     }
 
-    #endregion
+    class AlterStatementChecker
+    {
+        public static void OnCodeElement(AlterStatement statement, CodeElementsParser.AlterStatementContext context)
+        {
+#if EUROINFO_RULES
+            DiagnosticUtils.AddError(statement, "ALTER should not be used", context);
```

Comment:
For EUROINFO_RULES, you can also generate a warning.

Reply:
done